### PR TITLE
[ODS-4199] Postman PUT/POST request tests for API clients with multiple Edorg types

### PR DESCRIPTION
**Pull request by semalaiappan**

### CODE REVIEW

File: Postman Test Suite/Ed-Fi ODS-API Integration Test Suite Multiple Edorg types.postman_collection.json [hunk 1]
```diff
@@ -0,0 +1,1960 @@
...
+									""
+								]
+							}
+						}
+					],
+					"protocolProfileBehavior": {},
+					"_postman_isSubFolder": true
+				},
+				{
+					"name": "Scenario 6  Create a DisciplineIncident (associated with the different SchoolId) ",
```
Comment (tbmeador-dlp) [inline Postman Test Suite/Ed-Fi ODS-API Integration Test Suite Multiple Edorg types.postman_collection.json:1304]: For the scenarios where we are testing that we can not create a resourse, does it make sense to include "Can Not Create..." as part of the name?

File: Postman Test Suite/Ed-Fi ODS-API Integration Test Suite Multiple Edorg types.postman_collection.json [hunk 2]
```diff
@@ -0,0 +1,1960 @@
...
+								"{{ApiBaseUrl}}"
+							],
+							"path": [
+								"data",
+								"v3",
+								"ed-fi",
+								"communityProviders"
+							]
+						},
+						"description": "This api post method adds new academicWeeks for particular school .\nThis test method will throw WeekIdentifier is required error when WeekIdentifier is not passed"
```
[the remark this answers is not in the document]
Comment (semalaiappan) [reply Postman Test Suite/Ed-Fi ODS-API Integration Test Suite Multiple Edorg types.postman_collection.json]: yes , I updated . Thanks !

File: Postman Test Suite/Ed-Fi ODS-API Integration Test Suite Multiple Edorg types.postman_collection.json [hunk 3]
```diff
@@ -0,0 +1,1960 @@
...
+									"listen": "test",
+									"script": {
+										"id": "5a9d7b7c-8274-4f5d-b97b-54cb8b2066e1",
+										"exec": [
+											"pm.test(\"Status code is 200\", () => {",
+											"    pm.expect(pm.response.code).to.equal(200);",
+											"});",
+											"const __ = require('lodash');",
+											"const responseItems = pm.response.json();",
+											"pm.environment.set('known:staffUniqueId', __.first(responseItems).studentUniqueId);"
```
[the remark this answers is not in the document]
Comment (semalaiappan) [reply Postman Test Suite/Ed-Fi ODS-API Integration Test Suite Multiple Edorg types.postman_collection.json]: yes , I updated . Thanks !

File: Postman Test Suite/Ed-Fi ODS-API Integration Test Suite Multiple Edorg types.postman_collection.json (hunk 1 above)
Comment (semalaiappan) [reply Postman Test Suite/Ed-Fi ODS-API Integration Test Suite Multiple Edorg types.postman_collection.json]: yes , I updated . Thanks !

**Comment by semalaiappan** [comment]
@tbmeador-dlp .I fixed all review comments and ready for review again . thank  you !

**Comment by tbmeador-dlp** [comment]
@semalaiappan Your branch builds are broken. Do you need to rebase maybe?

**Comment by semalaiappan** [comment]
@tbmeador-dlp  All builds are green now

File: Postman Test Suite/Ed-Fi ODS-API Integration Test Suite Multiple Edorg types.postman_collection.json [hunk 4]
```diff
@@ -0,0 +1,1944 @@
...
+											"key": "schoolId",
+											"value": "{{known:schoolId}}"
+										}
+									]
+								}
+							},
+							"response": []
+						},
+						{
+							"name": "Update studentProgramAssociations",
```
Comment (tbmeador-dlp) [inline Postman Test Suite/Ed-Fi ODS-API Integration Test Suite Multiple Edorg types.postman_collection.json:600]: I see in your other scenarios you're doing a GET to verify your PUT/POST data went in. Should we be consistent and add one for this scenario also?

File: Postman Test Suite/Ed-Fi ODS-API Integration Test Suite Multiple Edorg types.postman_collection.json [hunk 5]
```diff
@@ -0,0 +1,1944 @@
...
+									""
+								]
+							}
+						}
+					],
+					"protocolProfileBehavior": {},
+					"_postman_isSubFolder": true
+				},
+				{
+					"name": "Scenario 3 Create a StudentProgramAssociation for a non-associated EducationOrganizationId value",
```
Comment (tbmeador-dlp) [inline Postman Test Suite/Ed-Fi ODS-API Integration Test Suite Multiple Edorg types.postman_collection.json:692]: This looks like another "Cannot" scenario. Can we update the name?

File: Postman Test Suite/Ed-Fi ODS-API Integration Test Suite Multiple Edorg types.postman_collection.json (hunk 4 above)
Comment (semalaiappan) [reply Postman Test Suite/Ed-Fi ODS-API Integration Test Suite Multiple Edorg types.postman_collection.json:600]: Reason is Get is not implemented for multiple EdOrg types . There is another story ODS-4198 for implementation and another story for postman scripts ODS-4248

File: Postman Test Suite/Ed-Fi ODS-API Integration Test Suite Multiple Edorg types.postman_collection.json (hunk 5 above)
Comment (semalaiappan) [reply Postman Test Suite/Ed-Fi ODS-API Integration Test Suite Multiple Edorg types.postman_collection.json]: yes , Fixed it . Thanks

**Pull request merged**